packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.styles.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  ITooltipStyleProps,
+  ITooltipStyles,
+  TooltipDelay
+} from './Tooltip.types';
+import { AnimationClassNames } from '../../Styling';
+
+export const getStyles = (props: ITooltipStyleProps): ITooltipStyles => {
+  const { className, delay, maxWidth, theme } = props;
+  const { palette, fonts } = theme;
+
+  return {
+    root: [
+      'ms-Tooltip',
+      className,
```

Comment:
the className should go at the end of the array so that it has precedence over the default rules.